unlock-protocol-com/src/components/layout/DefaultLayout.tsx

```diff
@@ -1,17 +1,119 @@
+import { Footer } from '@unlock-protocol/ui'
 import { ReactNode } from 'react'
-import Footer from '../interface/Footer'
+import { EMAIL_SUBSCRIPTION_FORM } from '../../config/constants'
+import { useMembership } from '../../hooks/useMembership'
 import { Navigation } from '../interface/Navigation'
 
 interface Props {
   children?: ReactNode | ReactNode[]
 }
 
 export function Layout({ children }: Props) {
+  const { becomeMember } = useMembership()
+
+  const FOOTER = {
+    subscriptionForm: {
+      description: 'Sign up for updates & fresh news about Unlock.',
+      onSubmit: async (email: string) => {
+        const { portalId, formGuid } = EMAIL_SUBSCRIPTION_FORM
+        const endpoint = `https://api.hsforms.com/submissions/v3/integration/submit/${portalId}/${formGuid}`
+        const options = {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+            fields: [
+              {
+                name: 'email',
+                value: email,
+              },
+            ],
+          }),
+        }
+
+        await fetch(endpoint, options)
+      },
+    },
+    logoUrl: 'https://unlock-protocol.com/',
+    privacyUrl: '/privacy',
+    termsUrl: '/terms',
+    actions: [
+      {
+        label: 'Launch App',
+        url: 'https://staging-app.unlock-protocol.com/locks',
```

Comment:
Not right! Please use env vars here to distinguish links between staging and prod